libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -139,7 +139,8 @@ bool ASTJsonConverter::visit(TypeName const&)
 
 bool ASTJsonConverter::visit(ElementaryTypeName const& _node)
 {
-	addJsonNode("ElementaryTypeName", { make_pair("name", Token::toString(_node.typeName())) });
+	string name = Type::fromElementaryTypeName(_node.typeName())->toString();
```

Comment:
there is one. I just chose this method because it seemed easier at the time. But I'll change it.